nix/nix.go

```diff
@@ -241,6 +241,7 @@ func Push(ctx *ssh.SSHContext, host Host, paths ...string) (err error) {
 	for _, path := range paths {
 		cmd := exec.Command(
 			"nix", "copy",
+			"--substitute-on-destination",
```

Comment:
Can we use the short flag instead? `-s` .. Because.. `--substitute-on-destination` is neither compatible with Nix v2.2 or 2.3. (ref: https://github.com/NixOS/nix/commit/aa739e783993864aa6e0c8a4820e6b59f4626d92)

Reply:
I usually dislike using short opts since it makes it harder to understand the code but in this case it is probably warranted due to the breaking change in nix.